lib/routes/account.js

```diff
@@ -1231,111 +1227,6 @@ module.exports = function (
     },
     {
       method: 'POST',
-      path: '/account/unlock/resend_code',
```

Comment:
One suggestion would be, to keep these routes as stubs and have them log an error, and return a `410 Gone`.  This will help us uncover whether any unexpected use of them is occurring in practice.